### ISSUE TITLE
Benign, seemingly random "Cipher is not initialized" errors

### ISSUE THREAD

**Issue by beaumartinez** [dhleong/ps4-waker#31]
I often get—

```
/usr/local/lib/node_modules/ps4-waker/lib/packets.js:109
        throw new Error("Cipher is not initialized!");
        ^

Error: Cipher is not initialized!
    at Packet.encrypted (/usr/local/lib/node_modules/ps4-waker/lib/packets.js:109:15)
    at Ps4Socket.login (/usr/local/lib/node_modules/ps4-waker/lib/ps4socket.js:246:6)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
```

Once `ps4-waker` has woken my PS4 up. It's not really a problem (since the PS4 wakes up fine), but it still seems odd to me.

**Comment by dhleong** [comment]
It looks like it's trying to log you in when the error occurs. I assume when it's all fine it turns on and takes you into your account, but when you get this error it stays at the "Press the PS button" screen?

**Comment by dhleong** [comment]
I believe this was fixed by the above commit (and/or 2811fb562b961c406cf483d6bd8a2e7c89132680). Please comment with details if you encounter it again!
